pom.xml

```diff
@@ -1423,34 +1452,12 @@
             </goals>
             <configuration>
               <includeScope>test</includeScope>
-              <outputFile>${test_classpath_file}</outputFile>
+	      <outputProperty>test_classpath</outputProperty>
             </configuration>
           </execution>
         </executions>
       </plugin>
 
-      <!-- This plugin reads a file into maven property. And it lets us write groovy !! -->
```

Comment:
I was not aware of this back then, did it even exist in the 2.9 version of maven dependency plugin ? Anyway the proof that this change is legit is that - it passes the maven test build especially the distributed ones in core(and there are lots of them).